src/usecases/usecases.go

```diff
@@ -0,0 +1,134 @@
+package usecases
+
+import (
+	"fmt"
+
+	"game-tracker/src/domain"
+)
+
+type UserRepository interface {
+	Store(user User)
+	FindById(id int) User
+	StoreInfo(user User, info string)
+	LoadInfo(user User) string
+}
+
+type LibraryRepository interface {
+	Store(library Library)
+	FindById(id int) Library
+}
+
+type User struct {
+	Id           int
+	PersonalInfo string
+	Player       domain.Player
+}
+
+type Library struct {
+	Id     int
+	Player domain.Player //belongs to player A
+	Games  []domain.Game
+}
+
+type Logger interface {
+	Log(message string) error
+}
+
+type ProfileInteractor struct {
+	UserRepository    UserRepository
+	LibraryRepository LibraryRepository
+	GameRepository    domain.GameRepository
+	Logger            Logger
+}
+
+func (interactor *ProfileInteractor) Profile(userId, libraryId int) (string, []domain.Game, error) {
+	var games []domain.Game
+	user := interactor.UserRepository.FindById(userId)
```

Comment:
This method is not good. What if your SQL query has some problems? `FindById` should return an error.